config/environments/production.rb

```diff
@@ -10,6 +10,8 @@
   # Rake tasks automatically ignore this option for performance.
   config.eager_load = true
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = { :host => 'unitern.herokuapp.com', :protocol => 'https' }
```

Comment:
Set the host via an ENV variable as it could change depending on the domain. For example you will be using environment as production on your staging too.